packages/integrations/node/CHANGELOG.md

```diff
@@ -1,5 +1,125 @@
 # @astrojs/node
 
+## 2.0.0
+
+### Major Changes
+
+- [#5056](https://github.com/withastro/astro/pull/5056) [`e55af8a23`](https://github.com/withastro/astro/commit/e55af8a23233b6335f45b7a04b9d026990fb616c) Thanks [@matthewp](https://github.com/matthewp)! - # Standalone mode for the Node.js adapter
+
+  New in `@astrojs/node` is support for **standalone mode**. With standalone mode you can start your production server without needing to write any server JavaScript yourself. The server starts simply by running the script like so:
+
+  ```shell
+  node ./dist/server/entry.mjs
+  ```
+
+  To enable standalone mode, set the new `mode` to `'standalone'` option in your Astro config:
+
+  ```js
+  import { defineConfig } from 'astro/config';
+  import nodejs from '@astrojs/node';
+
+  export default defineConfig({
+    output: 'server',
+    adapter: nodejs({
+      mode: 'standalone',
+    }),
+  });
+  ```
+
+  See the @astrojs/node documentation to learn all of the options available in standalone mode.
+
+  ## Breaking change
+
+  This is a semver major change because the new `mode` option is required. Existing @astrojs/node users who are using their own HTTP server framework such as Express can upgrade by setting the `mode` option to `'middleware'` in order to build to a middleware mode, which is the same behavior and API as before.
+
+  ```js
+  import { defineConfig } from 'astro/config';
+  import nodejs from '@astrojs/node';
+
+  export default defineConfig({
+    output: 'server',
+    adapter: nodejs({
+      mode: 'middleware',
+    }),
+  });
+  ```
+
+### Minor Changes
+
+- [#5056](https://github.com/withastro/astro/pull/5056) [`e55af8a23`](https://github.com/withastro/astro/commit/e55af8a23233b6335f45b7a04b9d026990fb616c) Thanks [@matthewp](https://github.com/matthewp)! - # Adapter support for `astro preview`
+
+  Adapters are now about to support the `astro preview` command via a new integration option. The Node.js adapter `@astrojs/node` is the first of the built-in adapters to gain support for this. What this means is that if you are using `@astrojs/node` you can new preview your SSR app by running:
+
+  ```shell
+  npm run preview
+  ```
+
+  ## Adapter API
+
+  We will be updating the other first party Astro adapters to support preview over time. Adapters can opt-in to this feature by providing the `previewEntrypoint` via the `setAdapter` function in `astro:config:done` hook. The Node.js adapter's code looks like this:
```

Comment:
```suggestion
  We will be updating the other first party Astro adapters to support preview over time. Adapters can opt in to this feature by providing the `previewEntrypoint` via the `setAdapter` function in `astro:config:done` hook. The Node.js adapter's code looks like this:
```
I'm not sure whether I'm supposed to review this PR, but just in case, I made a correction!